cmd/tapcli/assets.go

```diff
@@ -261,7 +261,7 @@ var finalizeBatchCommand = cli.Command{
 		},
 		cli.Uint64Flag{
 			Name: feeRateName,
-			Usage: "if set, the fee rate in sat/kw to use for " +
+			Usage: "if set, the fee rate in sat/vB to use for " +
```

Comment:
This is correct, but the RPC should still be sat/kw. So we need to do the conversion within the CLI as a convenience to the user.